frontend/components/auth/google.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Link, withRouter } from 'react-router-dom';
+import { GoogleLogin } from 'react-google-login';
+import PropTypes from 'prop-types';
+
+
+class GoogleAuth extends React.Component {
+    constructor(props) {
+        super(props);
```

Comment:
https://tylermcginnis.com/functional-components-vs-stateless-functional-components-vs-stateless-components/ no need to build a full-fledged component just for your learning - dont need to update this